tests/integration/aval/aval-tests.yml

```diff
@@ -36,6 +36,18 @@ build-aval-docker:
     - cd tests/integration && python /aval/main.py --delegation-config $DELEGATION_CONFIG
       --run-before-on-host "./run_all.sh --device --machine $TCB_MACHINE --report
       --tcb-custom-image ${CI_REGISTRY_IMAGE}/${IMAGE_NAME}:${GITLAB_DOCKERREGISTRY_SUFFIX_LATEST}"
+      # workaround for cloud credentials bug when using TCB deploy
```

Comment:
The comment doesn't explain the bug properly - it's not related to the cloud credentials. I suggest something like "to circumvent the mismatch between Aktualizr's database and ostree, install a different version than the one Aktualizr thinks it's currently installed (the one before tcb deploy), which externally forces both Aktualizr and ostree to sync. In particular, if we're running a nightly, install a release build and vice-versa." or something.

Reply:
Done! Thanks!